docs/api/components/schemas.yaml

```diff
@@ -1273,7 +1273,7 @@ Database:
         db_state_changed:
           $ref: '#/Event-config'
     feed_type:
-      description: The type of feed to use to communicate with Couchbase Server.
+      description: The type of feed to use to communicate with Couchbase Server. This will use DCP regardless of specification.
```

Comment:
Are we required to keep TAP in the enum for backward compatibility, or could it be removed.

Reply:
I can definitely just remove this.